ush/forecast_postdet.sh

```diff
@@ -145,6 +145,11 @@ EOF
     ${NLN} "${FIXgfs}/orog/${CASE}/${CASE}.mx${OCNRES}_oro_data.tile${n}.nc" "${DATA}/INPUT/oro_data.tile${n}.nc"
     ${NLN} "${FIXgfs}/orog/${CASE}/${CASE}_grid.tile${n}.nc"     "${DATA}/INPUT/${CASE}_grid.tile${n}.nc"
   done
+  if [[ "${DO_NEST:-NO}" == YES ]] ; then
```

Comment:
@SamuelTrahanNOAA For record, we will need similar changes for warm_start. Will it be the same as changes in cold_start?

Reply:
Never mind, I looked at wrong codes.